usecases/objects/validation/properties_validation.go

```diff
@@ -59,7 +59,10 @@ func (v *Validator) properties(ctx context.Context, object interface{}, class *m
 		return nil
 	}
 
-	inputSchema := isp.(map[string]interface{})
+	inputSchema, ok := isp.(map[string]interface{})
+	if !ok {
+		return fmt.Errorf("object's properties defined wrong")
```

Comment:
could we add the input schema variable to the error? So users can see what could be wrong?

Reply:
And maybe reformulate the error message to something like "could not recognize objects properties: %v"?

Reply:
I will change that to `could not recognize objects properties`

Reply:
@dirkkul fixed! thanks for the comment